DeviceManager/DeviceManager/View/Date+Extension.swift

```diff
@@ -21,4 +21,13 @@ extension Date {
         }
         return dateDouble
     }
+    
+    static func getStringFromTimeStamp (timeStamp: Double , format: String? = "hh:mm a MMM dd YYYY") -> String? {
```

Comment:
take the format as a global variable similar to `defaultDateFormat` and use it here